dicers_bot/bot.py

```diff
@@ -390,9 +390,7 @@ def show_users(self, chat_id: str) -> Optional[Message]:
 
         message = "\n".join([str(user) for user in chat.users])
 
-        if message:
-            return self.updater.bot.send_message(chat_id=chat_id, text=message)
-        else:
-            self.logger.info("No users to show.")
+        if not message:
+            message = "No active users, the user needs to write a message in the chat to be activated (not just a command)"
```

Comment:
Maybe change the wording? The current wording suggest the user himself is being activated by writing a message, when in reality the user is "active" anyway, he's just not in our list.

```suggestion
            message = "No active users. Users need to write a message in the chat to be recognized (not just a command)"
```

Reply:
My main concern here is that users will spam the chat after this message because they think the bot will ignore them in events otherwise.